index.js

```diff
@@ -281,7 +281,17 @@ function redisStore(args) {
       if (err) {
         return cb && cb(err);
       }
-      conn.del(key, handleResponse(conn, cb));
+
+      if (Object.prototype.toString.call(key) === '[object Array]') {
+        var multi = conn.multi();
+        for (var i = 0, l = key.length; i < l; ++i) {
```

Comment:
It's a bit faster using a local variable than accessing an object property. I do most of my loops this way. The fastest is actually something like:
```
var i = key.length();
while (i--) {
  multi.del(key[i]);
}
```
With the while loop, you're no longer having to test `i` against the length on every loop. But these are micro-optimizations most people will never notice. If you'd rather get rid of the `l` variable, we can do that.